react-app/src/library/services/vendor-implementations/jabra/jabra.ts

```diff
@@ -184,9 +185,31 @@ export default class JabraService extends VendorImplementation {
         if (!this.callLock) {
             return;
         }
+        this.incomingConversationId = '';
         this.callControl.offHook(true);
     }
 
+    async rejectCall(): Promise<void> {
+        try {
+            if (this.callLock) {
+                this.callControl.ring(false);
+            } else {
+                this.logger.info('Currently not in possession of the Call Lock; Cannot react to Device Actions')
+            }
+            this.callControl.releaseCallLock();
```

Comment:
this seems like this should be in the `if` block. I'd maybe rewrite this like this:
```
if (!this.callLock) {
  return this.logger.info('Currently not in possession of the Call Lock; Cannot react to Device Actions');
}

this.callControl.ring(false);
this.callControl.releaseCallLock();
```

Reply:
also, does `releaseCallLock` need to be awaited?

Reply:
I think it would make the most sense to ensure it is done properly or if we get the right error message.  But looking through their docs, they don't `await` any of the examples of `releaseCallLock`